publish/stack-nodejs-express/stack-nodejs-express.md

```diff
@@ -92,44 +94,35 @@ NAME        URL
 *incubator https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
 ```
 
-Next, run the following command to add the URL for your stack configuration file:
+Next, run the following command to add the URL for the public Kabanero stack hub:
 
 ```
-appsody repo add <my-org-stack> <URL>
+appsody repo add kabanero https://github.com/kabanero-io/kabanero-stack-hub/releases/download/0.7.0/kabanero-stack-hub-index.yaml
```

Comment:
I saw your remark about using a specific version versus 0.7.0.
It is pretty safe to use `latest` for this particuluar tutorial.

Reply:
I would think having a fixed known working version would be the best solution. It would not be good to use latest and for some changes to go not a stack that causes the steps in the guide to fail.